plannerbenchmark/postProcessing/helpers.py

```diff
@@ -0,0 +1,77 @@
+from plannerbenchmark.postProcessing.metrics import (
```

Comment:
Why are the following metrics not included in this implementation? Is this implementation only a proof of concept now and you plan to extend it for the below metrics later?
- DynamicClearanceMetric
- IntegratedErrorMetric
- SelfClearanceMetric

The above metrics are instantiated for different types of robot in the current version of the postprocessor .